### PR TITLE
WF-63198-Add code snippet to create a waterfall chart in UG

### PR DESCRIPTION
**Pull request by Suriya-Balamurugan**
Kindly review the below merge request and let me know your feedback.

<table>
<tr>
<td>
Task Link
</td>
<td>
<a href="https://syncfusion.atlassian.net/browse/WF-63198">https://syncfusion.atlassian.net/browse/WF-63198</a>
</td>
</tr>
<tr>
<td>
Description
</td>
<td>
Add code snippet to create a waterfall chart in UG.
</td>
</tr>
<tr>
<td>
Solution
</td>
<td>
1. Added code snippet to create waterfall chart in Word document in all platforms. <br/>
2. Added output Word document screenshot.<br/>
3. Added description and notes about the waterfall chart.
</td>
</tr>
</table>

### CODE REVIEW

File: File-Formats/DocIO/Working-with-Charts.md [hunk 1]
```diff
@@ -2382,6 +2382,270 @@ You can download a complete working sample from [GitHub](https://github.com/Sync
 N> 1. To convert chart in Word document as image, it is need to refer chart conversion related [assemblies](https://help.syncfusion.com/file-formats/docio/assemblies-required#converting-charts) or [NuGet packages](https://help.syncfusion.com/file-formats/docio/nuget-packages-required#converting-charts).
 N> 2. The ChartToImageConverter is supported from .NET Framework 4.0 onwards.
 
+## Word 2016 Charts
+
+Essential DocIO supports creating and manipulating new and modern chart types such as waterfall, which are introduced in Microsoft Word 2016.
```
Comment (MohanaselvamJothi) [inline File-Formats/DocIO/Working-with-Charts.md:2387]: Which is

File: File-Formats/DocIO/Working-with-Charts.md [hunk 2]
```diff
@@ -2382,6 +2382,270 @@ You can download a complete working sample from [GitHub](https://github.com/Sync
...
+
+### Waterfall
+
+[Waterfall](https://support.office.com/en-us/article/Create-a-waterfall-chart-in-Office-2016-8de1ece4-ff21-4d37-acd7-546f5527f185#) chart helps understand the finances of business owners by viewing profit and loss statements. You can quickly illustrate the line items in your financial data and get a clear picture of how each item is impacting your bottom line using a Waterfall chart. Refer to the following code to create a Waterfall chart.
```
Comment (MohanaselvamJothi) [inline File-Formats/DocIO/Working-with-Charts.md:2391]: Refer to the following code to create a Waterfall chart. 

Please modify this line

**Review by MohanaselvamJothi — CHANGES_REQUESTED** [review]

File: File-Formats/DocIO/Working-with-Charts.md (hunk 2 above)
Comment (Suriya-Balamurugan) [reply File-Formats/DocIO/Working-with-Charts.md]: Modified

File: File-Formats/DocIO/Working-with-Charts.md (hunk 1 above)
Comment (Suriya-Balamurugan) [reply File-Formats/DocIO/Working-with-Charts.md]: Modified

**Review by MohanaselvamJothi — APPROVED** [review]

**Review by MohanaselvamJothi — APPROVED** [review]

**Pull request merged**